src/components/activities/pharmacyActivity/pharmaceutical/components/forms/pharmaceuticalForm/consts.ts

```diff
@@ -2,39 +2,58 @@ import { MedicalDTO } from "generated";
 import { z } from "zod";
 import { TFormValues } from "./types";
 
-export const MedicalDTOSchema = z.object({
-  prodCode: z.string({
-    error: "code is required",
-  }),
+export type MedicalErrorKey =
+  | "pharmacy.form.errors.prodCodeRequired"
+  | "pharmacy.form.errors.typeRequired"
+  | "pharmacy.form.errors.descriptionRequired"
+  | "pharmacy.form.errors.pcsperpckRequired"
+  | "pharmacy.form.errors.minqtyRequired";
```

Comment:
Why this ? Wheve already have LocaleKey in the code, related to the custom useTranslation hook that you're using...